src/sentry/static/sentry/app/components/searchBar.tsx

```diff
@@ -103,11 +105,15 @@ class SearchBar extends React.PureComponent<Props, State> {
             />
             <StyledIconSearch className="search-input-icon" size="sm" color="gray500" />
             {this.state.query !== this.props.defaultQuery && (
-              <div>
-                <a className="search-clear-form" onClick={this.clearSearch}>
-                  <IconClose />
-                </a>
-              </div>
+              <SearchClearForm
+                type="button"
```

Comment:
I don't think that you need to add this type here

Reply:
its inside a form, otherwise hitting enter clears the form.